test/integration/ComputeFlow.test.ts

```diff
@@ -12,6 +12,15 @@ import { homedir } from 'os'
 import fs from 'fs'
 import { ProviderFees, Erc20CreateParams } from '../../src/@types'
 
+/**
+ * Simple blocking sleep function
+ */
+async function sleep(ms: number) {
```

Comment:
Aquarius class has a `sleep` method so rather use that here instead of duplicating same code? Sounds like that `sleep` method should not live on Aquarius class though